### ISSUE TITLE
intallation error

### ISSUE THREAD

**Issue by filipgoc** [BillyBlaze/OctoPrint-FullScreen#4]
I'm trying out the new 0.14 RC1 Octopi. 

I'm getting this error on installation: 
```
Something went wrong
Installing the plugin from URL "unknown" failed: Could not parse output from pip
```

Ideas? 
Below a full log is attached. 

```
Installing plugin "Fullscreen Webcam" from https://github.com/BillyBlaze/OctoPrint-FullScreen/archive/master.zip...
Installation needs to process external dependencies, that might make it take a bit longer than usual depending on the pip version
/home/pi/oprint/bin/python -m pip install https://github.com/BillyBlaze/OctoPrint-FullScreen/archive/master.zip
Collecting https://github.com/BillyBlaze/OctoPrint-FullScreen/archive/master.zip
Downloading https://github.com/BillyBlaze/OctoPrint-FullScreen/archive/master.zip
Requirement already satisfied (use --upgrade to upgrade): OctoPrint-Fullscreen==0.0.1 from https://github.com/BillyBlaze/OctoPrint-FullScreen/archive/master.zip in /home/pi/oprint/lib/python2.7/site-packages
Requirement already satisfied: OctoPrint in /home/pi/oprint/lib/python2.7/site-packages/OctoPrint-1.3.2-py2.7.egg (from OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: flask<0.11,>=0.9 in /home/pi/oprint/lib/python2.7/site-packages/Flask-0.10.1-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: Jinja2<2.9,>=2.8 in /home/pi/oprint/lib/python2.7/site-packages/Jinja2-2.8.1-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: werkzeug<0.9,>=0.8.3 in /home/pi/oprint/lib/python2.7/site-packages/Werkzeug-0.8.3-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: tornado==4.0.2 in /home/pi/oprint/lib/python2.7/site-packages/tornado-4.0.2-py2.7-linux-armv7l.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: sockjs-tornado<1.1,>=1.0.2 in /home/pi/oprint/lib/python2.7/site-packages/sockjs_tornado-1.0.3-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: PyYAML<3.11,>=3.10 in /home/pi/oprint/lib/python2.7/site-packages/PyYAML-3.10-py2.7-linux-armv7l.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: Flask-Login<0.3,>=0.2.2 in /home/pi/oprint/lib/python2.7/site-packages/Flask_Login-0.2.11-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: Flask-Principal<0.4,>=0.3.5 in /home/pi/oprint/lib/python2.7/site-packages/Flask_Principal-0.3.5-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: Flask-Babel<0.10,>=0.9 in /home/pi/oprint/lib/python2.7/site-packages/Flask_Babel-0.9-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: Flask-Assets<0.11,>=0.10 in /home/pi/oprint/lib/python2.7/site-packages/Flask_Assets-0.10-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: markdown<2.7,>=2.6.4 in /home/pi/oprint/lib/python2.7/site-packages/Markdown-2.6.8-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: pyserial<2.8,>=2.7 in /home/pi/oprint/lib/python2.7/site-packages/pyserial-2.7-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: netaddr<0.8,>=0.7.17 in /home/pi/oprint/lib/python2.7/site-packages/netaddr-0.7.19-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: watchdog<0.9,>=0.8.3 in /home/pi/oprint/lib/python2.7/site-packages/watchdog-0.8.3-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: sarge<0.2,>=0.1.4 in /home/pi/oprint/lib/python2.7/site-packages/sarge-0.1.4-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: netifaces<0.11,>=0.10 in /home/pi/oprint/lib/python2.7/site-packages/netifaces-0.10.5-py2.7-linux-armv7l.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: pylru<1.1,>=1.0.9 in /home/pi/oprint/lib/python2.7/site-packages/pylru-1.0.9-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: rsa<3.3,>=3.2 in /home/pi/oprint/lib/python2.7/site-packages/rsa-3.2.3-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: pkginfo<1.3,>=1.2.1 in /home/pi/oprint/lib/python2.7/site-packages/pkginfo-1.2.1-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: requests<2.8,>=2.7 in /home/pi/oprint/lib/python2.7/site-packages/requests-2.7.0-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: semantic_version<2.5,>=2.4.2 in /home/pi/oprint/lib/python2.7/site-packages/semantic_version-2.4.2-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: psutil<3.3,>=3.2.1 in /home/pi/oprint/lib/python2.7/site-packages/psutil-3.2.2-py2.7-linux-armv7l.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: Click<6.3,>=6.2 in /home/pi/oprint/lib/python2.7/site-packages/click-6.2-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: awesome-slugify<1.7,>=1.6.5 in /home/pi/oprint/lib/python2.7/site-packages/awesome_slugify-1.6.5-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: feedparser<5.3,>=5.2.1 in /home/pi/oprint/lib/python2.7/site-packages/feedparser-5.2.1-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: chainmap<1.1,>=1.0.2 in /home/pi/oprint/lib/python2.7/site-packages/chainmap-1.0.2-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: future<0.16,>=0.15 in /home/pi/oprint/lib/python2.7/site-packages/future-0.15.2-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: scandir<1.4,>=1.3 in /home/pi/oprint/lib/python2.7/site-packages/scandir-1.3-py2.7-linux-armv7l.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: websocket-client<0.41,>=0.40 in /home/pi/oprint/lib/python2.7/site-packages/websocket_client-0.40.0-py2.7.egg (from OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: itsdangerous>=0.21 in /home/pi/oprint/lib/python2.7/site-packages/itsdangerous-0.24-py2.7.egg (from flask<0.11,>=0.9->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: MarkupSafe in /home/pi/oprint/lib/python2.7/site-packages/MarkupSafe-1.0-py2.7-linux-armv7l.egg (from Jinja2<2.9,>=2.8->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: certifi in /home/pi/oprint/lib/python2.7/site-packages/certifi-2017.01.23-py2.7.egg (from tornado==4.0.2->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: backports.ssl_match_hostname in /home/pi/oprint/lib/python2.7/site-packages/backports.ssl_match_hostname-3.5.0.1-py2.7.egg (from tornado==4.0.2->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: blinker in /home/pi/oprint/lib/python2.7/site-packages/blinker-1.4-py2.7.egg (from Flask-Principal<0.4,>=0.3.5->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: Babel>=1.0 in /home/pi/oprint/lib/python2.7/site-packages/Babel-2.4.0-py2.7.egg (from Flask-Babel<0.10,>=0.9->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: speaklater>=1.2 in /home/pi/oprint/lib/python2.7/site-packages/speaklater-1.3-py2.7.egg (from Flask-Babel<0.10,>=0.9->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: webassets>=0.10 in /home/pi/oprint/lib/python2.7/site-packages/webassets-0.12.1-py2.7.egg (from Flask-Assets<0.11,>=0.10->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: argh>=0.24.1 in /home/pi/oprint/lib/python2.7/site-packages/argh-0.26.2-py2.7.egg (from watchdog<0.9,>=0.8.3->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: pathtools>=0.1.1 in /home/pi/oprint/lib/python2.7/site-packages/pathtools-0.1.2-py2.7.egg (from watchdog<0.9,>=0.8.3->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: pyasn1>=0.1.3 in /home/pi/oprint/lib/python2.7/site-packages/pyasn1-0.2.3-py2.7.egg (from rsa<3.3,>=3.2->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: regex in /home/pi/oprint/lib/python2.7/site-packages/regex-2017.02.08-py2.7-linux-armv7l.egg (from awesome-slugify<1.7,>=1.6.5->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: Unidecode<0.05,>=0.04.14 in /home/pi/oprint/lib/python2.7/site-packages/Unidecode-0.04.20-py2.7.egg (from awesome-slugify<1.7,>=1.6.5->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: six in /home/pi/oprint/lib/python2.7/site-packages/six-1.10.0-py2.7.egg (from websocket-client<0.41,>=0.40->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Requirement already satisfied: pytz>=0a in /home/pi/oprint/lib/python2.7/site-packages/pytz-2017.2-py2.7.egg (from Babel>=1.0->Flask-Babel<0.10,>=0.9->OctoPrint->OctoPrint-Fullscreen==0.0.1)
Done!
```

**Comment by foosel** [comment]
@filipgoc log says the plugin was already installed:

    Requirement already satisfied (use --upgrade to upgrade): OctoPrint-Fullscreen==0.0.1 from https://github.com/BillyBlaze/OctoPrint-FullScreen/archive/master.zip in /home/pi/oprint/lib/python2.7/site-packages

Out of curiousity, how did you trigger this install? Did you manually paste the URL into the "from URL" text box, or did you click the Install button next to the plugin the repository listing?

**Comment by filipgoc** [comment]
Thanks. Missed it in the stream of satisfactions. 

I clicked on the install button @foosel.

**Comment by foosel** [comment]
Weird. I wonder why it was already installed and why OctoPrint didn't properly detect that (should have been a "Reinstall" button in that case). In any case, I pushed a change today that will automatically detect situations like the above and force a reinstall of the plugin in such cases. Will be released with 1.3.3.

**Comment by BillyBlaze** [comment]
Awesome thank you both, closing this issue.
